src/main.rs

```diff
@@ -149,9 +144,24 @@ fn parse<T: BufRead>(
                 match serde_json::from_str(&line) {
                     Ok(event) => event,
                     Err(_) => {
-                        // Assume this line isn't part of the test result output. The test itself may
-                        // have printed e.g. a log starting with '{'
-                        continue;
+                        // Our logging inserts logs that look like json, but aren't. If we can't parse the line as json
```

Comment:
Code is fine, but a couple notes on the comment:
- Make it clear that this is a temporary internal-only change that we have no intention to merge back to the public cargo2junit (right?)
- Please elaborate on the first sentence for the benefit of someone reading this without the necessary context. Explicitly tell them it's because slog-json as of v<Blah> no longer writes whole lines while locking stdout (or whatever) and so log lines written by slog-json can be comingled with test output lines written by `cargo test` instead of one-log-per-line. The change here is temporary while we get a fix into slog-json.